mlflow/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghcr.io/mlflow/mlflow
```

Comment:
Should we pin image version? e.g. ghcr.io/mlflow/mlflow:v2.12.1

Reply:
Pinned to v2:12.1